ThingIFSDK/ThingIFSDK/Utils/JsonSerializable.swift

```diff
@@ -13,3 +13,8 @@ internal protocol JsonSerializable {
     func makeJson() -> [String : Any]
 
 }
+
+internal protocol JsonDeserializable {
```

Comment:
JsonSerializable /JsonDeserializable

The expected interface from the name is like this.
```
makeJson() -> String
init(_ fromJson: String)
```
It looks bit strange to me that those protocol treats [String : Any] as `json`.